src/Resources/Resources/help/New-AzPolicyExemption.md

```diff
@@ -0,0 +1,256 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.ResourceManager.dll-Help.xml
+Module Name: Az.Resources
+online version:
```

Comment:
Please fill in online version with : https://docs.microsoft.com/powershell/module/az.resources/{comlet-name}